src/components/portfolio/DetailsHeader.tsx

```diff
@@ -65,11 +68,9 @@ const DetailsHeader: React.FC<DetailsHeaderProps> = ({
             <ArrowBackIosIcon fontSize="large" />
           </IconButton>
         </div>
-        <div>
-          <span className={classes.title}>{name}</span>
-        </div>
+        <div>{name && <span className={classes.title}>{name}</span>}</div>
```

Comment:
Again, the use of && is not really clear for me.

Reply:
`name` may be undefined (during loading). In this case, the `<span>...</span>` element is not rendered.